org.eclipse.jdt.ls.tests/src/org/eclipse/jdt/ls/core/internal/BundleUtilsTest.java

```diff
@@ -63,9 +62,8 @@ public void testLoadWithWhitespace() throws BundleException, CoreException {
 		BundleContext context = JavaLanguageServerPlugin.getBundleContext();
 		Bundle installedBundle = context.getBundle(bundleLocation);
 		assertNotNull(installedBundle);
-		assertEquals(installedBundle.getState(), Bundle.INSTALLED);
+		assertEquals(installedBundle.getState(), Bundle.STARTING);
 
-		installedBundle.start();
 		String extResult = getBundleExtensionResult();
 		assertEquals("EXT_TOSTRING", extResult);

```

Comment:
Should this assertion not have failed before that current change?

Reply:
With start() (Which should not be called in test I must admit) without parameter, the assertion is passed.